Source/items.cpp

```diff
@@ -3410,18 +3410,18 @@ void ProcessItems()
 		auto &item = Items[ii];
 		if (!item._iAnimFlag)
 			continue;
-		item.AnimInfo.ProcessAnimation();
+		item.AnimInfo.processAnimation();
 		if (item._iCurs == ICURS_MAGIC_ROCK) {
-			if (item._iSelFlag == 1 && item.AnimInfo.CurrentFrame == 10)
-				item.AnimInfo.CurrentFrame = 0;
-			if (item._iSelFlag == 2 && item.AnimInfo.CurrentFrame == 20)
-				item.AnimInfo.CurrentFrame = 10;
+			if (item._iSelFlag == 1 && item.AnimInfo.getCurrentFrame() == 10)
+				item.AnimInfo.setCurrentFrame(0);
+			if (item._iSelFlag == 2 && item.AnimInfo.getCurrentFrame() == 20)
+				item.AnimInfo.setCurrentFrame(10);
 		} else {
-			if (item.AnimInfo.CurrentFrame == (item.AnimInfo.NumberOfFrames - 1) / 2)
+			if (item.AnimInfo.getCurrentFrame() == (item.AnimInfo.getNumberOfFrames() - 1) / 2)

```

Comment:
Might as well use isTriggerFrame here

Reply:
How exactly? it is kind of split number of frames in half. There is no setting TriggerFrame for items.

Reply:
Monster attack animations have a trigger frame where damage is applied and the associated sound effects are played, same for player attack animations.

Item drop animations play a sound effect on this frame, seems like a trigger frame to me.